bindings/rust/bench/README.md

```diff
@@ -16,7 +16,7 @@ mkdir .cargo
 cat aws-lc-config/s2n.toml > .cargo/config.toml
 cargo bench
 scripts/bench-memory.sh
-scripts/bench-past.sh
+cargo bench --bench handshake --bench throughput -- --profile-time 5
```

Comment:
Decided to remove historical benching from the quickstart and add flamegraphs, because historical benching erases the `target/criterion` folder results